### ISSUE TITLE
What is the license term of this project?

### ISSUE THREAD

**Issue by sajithdilshan** [louisbros/simple-websocket-server#4]
Is this project available under MIT License?

**Comment by louisbros** [comment]
Yes, you're free to use and modify this code however you like, however it should be considered incomplete and is currently not a complete implementation of the WebSocket spec.
